src/main/java/springboot/Model/Instructor.java

```diff
@@ -1,13 +1,8 @@
-package Model;
+package springboot.Model;
 
-import javax.persistence.CascadeType;
-import javax.persistence.Entity;
-import javax.persistence.OneToMany;
-import javax.persistence.OneToOne;
-import java.util.ArrayList;
-import java.util.List;
+import javax.persistence.*;
 
-@Entity
```

Comment:
do we want to remove these? how is it done in the lab?

Reply:
also if we do want to remove it we should just delete it instead

Reply:
We will uncomment it eventually, once the repository for Instructor is created

Reply:
kk